GeoffreyBooth-proposal.md

```diff
@@ -0,0 +1,78 @@
+# Proposal for the Future of CoffeeScript
+
+Here’s my suggestion for the way forward for CoffeeScript.
+
+## The Problem
+
+It’s important to define what problem(s) we’re trying to solve. In my opinion, it boils down to one: CoffeeScript is losing marketshare. It becomes harder to choose CoffeeScript for a project when fewer people use it, and when there isn’t public support for the language and a community committed to supporting it as the JavaScript ecosystem evolves. Choosing CoffeeScript for a project can also be stymied if another essential piece of the project, especially its framework, is incompatible with CoffeeScript in some way.
+
+Many people were drawn to CoffeeScript because it offered features that JavaScript once lacked: classes, destructuring, fat-arrow functions, etc. Once ES2015 arrived and JavaScript (mostly) caught up, those people abandoned CoffeeScript back for the JavaScript they were more familiar with. Those people aren’t coming back, nor should we pursue them. With ECMAScript proposals and development proceeding at a rapid pace, we stand no chance of out-innovating the JavaScript community in inventing new language features.
+
+Many other people, however, came to CoffeeScript for the clean, readable syntax and the many ways that the language itself helps prevent bugs, such as significant whitespace and the existential operator. CoffeeScript still has these advantages over all versions of JavaScript, and will retain them so long as ECMAScript strives for backward compatibility. But for CoffeeScript to remain a viable choice for developers, it must keep pace with the JavaScript community. At the very least it must be compatible with most popular frameworks and build tools; ideally it will also be current with the latest approved standard. The mantra of CoffeeScript is that “It’s just JavaScript”—but right now, JavaScript is ES2015.
```

Comment:
I like this block, however I want it to stress more of the importance of elegant and concise code.  I think a core tenant of the language is really about reducing unnecessary syntax whenever possible.  Making the code structure simple and powerful is a fundamental to the language.  This is something that should be a core value moving forward.